docs/pages/file-format-reference-overview.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Overview
+section: file-format
+chapter: Reference files
+permalink: /file-format/reference
+
+order: 101
+excerpt: Reference files overview
+---
+
+While the JSON encoded Sketch [File Format](/file-format) is both human and machine-readable, it is nevertheless something of a moving target since the underlying schema is subject to potential change with each new version of Sketch.
```

Comment:
> … is nevertheless something of a moving target …

As much as I like it from a wordsmith's perspective I'd prefer especially the opening excerpt to be more succinct. Many of the audience may speak English as their second or third language, something along to lines of _"… machine-readable, the format changes with new versions Sketch."_